spring/src/main/java/org/axonframework/spring/config/SpringAxonAutoConfigurer.java

```diff
@@ -187,6 +185,15 @@ public void registerBeanDefinitions(AnnotationMetadata importingClassMetadata, B
                     .getBeanDefinition());
         }
 
+        Optional<String> eventProcessorRegistry = findComponent(EventProcessorRegistry.class);
```

Comment:
This resembles the `eventHandlingConfiguration`   on line 181 better, so I'll assume this `EventProcessorRegistry` works in line with the old `EventHandlingConfiguration here. A.k.a., good stuff.

Reply:
Yes, I wanted to have `EventProcessorRegistry` as a bean in the application context for easier configuration.